pkg/containertools/test_files/expected_unpack/manifests/prometheusoperator.0.14.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,240 @@
+#! parse-kind: ClusterServiceVersion
```

Comment:
nit: in go, the idiomatic location for "ancillary test data" is the `testdata` directory; see https://golang.org/pkg/cmd/go/internal/test/